tools/Documentation/plugin-docs/metadata.md

```diff
@@ -152,3 +154,37 @@ sub get_tags_from_somewhere {
 
 1;
 ```
+
+### Converting existing plugins to named parameters
+
```

Comment:
I would mention some history here, as in that plugins used to rely on an ordered array before 0.9.3 and that has changed to named parameters.  

For the time being, it's also perfectly fine to remain on ordered array parameters if plugin devs don't want to update -- User settings won't be lost.